service/ucs.py

```diff
@@ -22,32 +21,33 @@ def login_get(headers):
             return handle.cookie
```

Comment:
I am thinking we should provide the existing cookie as well from _getUcsAuthInfo.

Reply:
I'm not sure if cookie info is realtime or not, becuase login function has never been called by rackhd.